File: unix/boot/spp/rpp/ratlibf/insub.f
      integer function insub (arg, file)
      integer arg (100), file (100)
      if (.not.(arg (1) .eq. 60 .and. arg (2) .ne. -2))goto 23000
      insub = 1
      call scopy (arg, 2, file, 1)
      goto 23001
23000 continue
      insub = 0
23001 continue
      return
      end

File: unix/boot/spp/rpp/ratlibr/insub.r
include	defs

# insub - determine if argument is STDIN substitution

   integer function insub (arg, file)
   character arg (ARB), file (ARB)

   if (arg (1) == LESS & arg (2) != EOS) {
      insub = YES
      call scopy (arg, 2, file, 1)
      }
   else
      insub = NO

   return
   end
